scene/2d/camera_2d.cpp

```diff
@@ -547,6 +547,7 @@ bool Camera2D::is_ignoring_rotation() const {
 void Camera2D::set_limit_enabled(bool p_limit_enabled) {
 	limit_enabled = p_limit_enabled;
 	_update_scroll();
+	emit_signal("_camera_limit_enabled_updated"); // Used for Camera2DEditorPlugin
```

Comment:
```suggestion
#ifdef TOOLS_ENABLED
	emit_signal("_camera_limit_enabled_updated");
#endif
```

Reply:
Marked as resolved, but I don't see it is actually resolved...